test/tangy-form_test.html

```diff
@@ -222,39 +222,39 @@
         test('hello world without errors', () => {
           const element = fixture('HelloWorldFixture');
           element.newResponse()
-          assert.equal(element.querySelector('tangy-form-item').shadowRoot.querySelector('#content').innerText, 'Hello world.');
+          assert.equal(element.querySelector('tangy-form-item').innerText, 'Hello world.');
         });
 
         test('hello world with caught errors in logic', () => {
           const element = fixture('ErrorInLogicFixture');
           element.newResponse()
           assert.equal(element.shadowRoot.querySelectorAll('.error').length, 5)
-          assert.equal(element.querySelector('tangy-form-item').shadowRoot.querySelector('#content').innerText, 'Hello world.');
+          assert.equal(element.querySelector('tangy-form-item').innerText, 'Hello world.');
         });
 
         test('hello world from template without errors', () => {
           const element = fixture('HelloWorldFromTemplateFixture');
           element.newResponse()
-          assert.equal(element.querySelector('tangy-form-item').shadowRoot.querySelector('#content').innerText, 'Hello world.');
+          assert.equal(element.querySelector('tangy-form-item').innerText, 'Hello world.');
         });
 
         test('should submit and be able to review an item', function() {
           let element = fixture('HelloWorldWithInputFixture')
           element.newResponse()
           element.querySelector('#item1').shadowRoot.querySelector('dom-if').render()
-          element.querySelector('#item1').shadowRoot.querySelector('[name=input1]').value = 'foo'
+          element.querySelector('#item1').querySelector('[name=input1]').value = 'foo'
           element.querySelector('#item1').shadowRoot.querySelector('#complete').click()
           element.querySelector('#item1').shadowRoot.querySelector('dom-if').render()
           element.querySelector('#item1').shadowRoot.querySelector('#open').click()
-          assert.equal(element.querySelector('#item1').shadowRoot.querySelector('[name="input1"]').value, 'foo')
+          assert.equal(element.querySelector('#item1').querySelector('[name="input1"]').value, 'foo')
```

Comment:
An added bonus of this approach is now input elements are always one level less deep.